src/material/select/select.md

```diff
@@ -66,6 +66,15 @@ If you want one of your options to reset the select's value, you can omit specif
 
 <!-- example(select-reset) -->
 
+### Allowing nullable options to be selected
+
+By default any options with a `null` or `undefined` value will reset the select's value. This is
```

Comment:
I'd probably remove the statement about mimicing native behavior, since I don't think that's the actual case anymore